python_modules/dagster/dagster/_core/definitions/asset_spec.py

```diff
@@ -67,6 +67,7 @@ def __new__(
                 Union[CoercibleToAssetKey, "AssetSpec", AssetsDefinition, SourceAsset, "AssetDep"]
             ]
         ] = None,
+        *,
```

Comment:
I think only `asset_key` should be before `*`, thoughts?

Reply:
Sounds good to me